client/src/components/clear-project-button.tsx

```diff
@@ -1,32 +1,25 @@
 "use client";
-import React, { useEffect } from "react";
+import React from "react";
 import { Button } from "./ui/button";
-import { clearActiveProject, get } from "~/app/db/db-actions";
 import { RefreshCwIcon } from "lucide-react";
 import { useRouter } from "next/navigation";
+import { useRxData } from "rxdb-hooks";
+import { type ActiveProject } from "~/lib/db/types";
 
 export default function ClearProjectButton() {
   const router = useRouter();
-  const [show, setShow] = React.useState<boolean>(false);
-  useEffect(() => {
-    get("activeProject")
-      .then(({ data }) => {
-        if (data) {
-          setShow(true);
-        }
-      })
-      .catch(() => {
-        //ignore
-      });
-  }, []);
+  const { result } = useRxData<ActiveProject>("activeProject", (collection) =>
```

Comment:
should it be async ? can we press the button before having the response from the local DB ?